ffmpeg-gpu/Dockerfile

```diff
@@ -0,0 +1,68 @@
+FROM nvidia/cuda:8.0-cudnn5-devel
+MAINTAINER Ryan Versaw <ryan.versaw@hudl.com>
+
+RUN apt-get update -y
```

Comment:
I'll change it up! Some of those were originally on the same line, but there were numerous separate `apt-get install` lines, so when I combined them, I left the `update` separate. I'll get them combined.